nativelib/src/main/scala/scala/scalanative/posix/sys/mman.scala

```diff
@@ -0,0 +1,132 @@
+package scala.scalanative
+package posix.sys
+
+import native._
+import types.{mode_t, off_t}
+
+@extern
+object mman {
+  def mmap(addr: Ptr[Byte],
+           len: CSize,
+           prot: CInt,
+           flags: CInt,
+           fd: CInt,
+           offset: off_t): Ptr[Byte] = extern
+
+  /* Deallocate any mapping for the region starting at ADDR and extending LEN
+   bytes.  Returns 0 if successful, -1 for errors (and sets errno).  */
```

Comment:
We don't include doc comments for bindings as the exact behavior might differ per platform.